dev/com.ibm.websphere.appserver.features/visibility/public/jdbcLegacyApi-1.0/resources/l10n/com.ibm.websphere.appserver.jdbcLegacyApi-1.0.properties

```diff
@@ -0,0 +1,20 @@
+###############################################################################
+# Copyright (c) 2013, 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature enables the legacy JDBC API.  This API allows the JDBC component to throw ConnectionWaitTimeoutException
```

Comment:
suggested edit:
This feature enables the traditional JDBC API. If a timeout occurs waiting for a JDBC connection, this API allows the JDBC component to throw a ConnectionWaitTimeoutException, instead of throwing a SQLTransientConnectionException with a ConnectionWaitTimeoutException as the cause. Use this feature only for compatibility with WebSphere Application 
Server traditional applications that require the ConnectionWaitTimeoutException to be thrown.

note that IBM style advises against using the term "legacy":
 For products that are still supported but are not being actively developed, use "existing," "heritage," "traditional," "established," "mature," "earlier," or another appropriate neutral description instead of "legacy."

Reply:
I added the "ID review not needed" because this pull is not intended for delivery.  It is just for Greg to experiment with.

Reply:
Hi David.  We are going to be pushing code changes out in this PR to support a new feature, but the feature will not be in OL.